### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by markgoddard**

### CODE REVIEW

File: ansible/roles/virtualbmc-daemon/tasks/main.yml [hunk 1]
```diff
@@ -1,4 +1,11 @@
 ---
+- name: Ensure package dependencies are installed
+  package:
+    name: "{{ item }}"
+    state: installed
+  with_items: "{{ vbmcd_packages }}"
```
Comment (w-miller) [inline ansible/roles/virtualbmc-daemon/tasks/main.yml]: Nit: can we use `loop` here to be more Ansible 2.6-ish?

Comment (markgoddard) [reply ansible/roles/virtualbmc-daemon/tasks/main.yml]: Just an old man stuck in his ways.

Coming right up

**Review by w-miller — APPROVED** [review]
Thanks for these!

**Comment by markgoddard** [comment]
"Tenks for your code"

**Pull request merged**